artifact-management/commands/gem-test.yml

```diff
@@ -1,36 +1,16 @@
-description: Perform the actual gem build
+description: Perform simple gem test
 parameters:
-  bundler_version:
-    description: The Bundler version that should be used for "bundle install" and the actual test command
-    type: string
-    default: 1.17.3
-  gem_repo_host:
```

Comment:
I think that `gem_repo_host` and `gem_repo_key` were copy-pasta (🍝 ) from a different command, so you're correct that they should be removed. But they should be removed in a separate commit with a message like "Removed unused parameters from gem-test" and not part of this monolithic commit. Again, to tell a cleaner story.